### ISSUE TITLE
Document the options that require a restart of NVDA to take effect

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#13685]
This issue follows [a discussion](https://nvda.groups.io/g/nvda/topic/visual_studio_lagging/90990627?p=,,,100,0,0,0::recentpostdate/sticky,,,100,2,0,90990627,previd%3D1652231726048659378,nextid%3D1649902511419872952&previd=1652231726048659378&nextid=1649902511419872952) on NVDA mailing-list.

### Steps to reproduce:
* Open NVDA advanced settings panel
* Check the checkbox "Enable selective registration for UI Automation events and property changes"
* Press F1 to read the documentation for this item
* Turn back to setting window and validate

This option has no effect until NVDA is restarted.

### Actual behavior:
The fact that this option takes effect only after NVDA restart is not documented in the user interface nor in the User Guide.

### Expected behavior:
* Document in the settings window and in the user guide  that this option requires to have NVDA restarted.
* Since the vast majority of the options in the settings window takes effect immediately when you press "OK" or "Apply", document the same way all the options (if any) that require NVDA to take effect.

#### NVDA installed/portable/running from source:
N/A
#### NVDA version:
2022.1beta4
#### Windows version:
N/A
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Same issue in previous versions including this option.
#### If NVDA add-ons are disabled, is your problem still occurring?
N/A
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
N/A

Cc @XLTechie, @Qchristensen who may be interested by the topic.
Cc @seanbudd, @feerrenrut to tag it as Good for new dev.

**Comment by lukaszgo1** [comment]
Isn't this a duplicate of #13505? While this issue is more general I'm not sure if there are other options where NVDA restart is required to apply them.

**Comment by CyrilleB79** [comment]
Yes this is definitely a duplicate of #13505?, where I had already commented by the way!
Closing here as duplicate.
